readthedocs/builds/views.py

```diff
@@ -97,7 +95,13 @@ def get_context_data(self, **kwargs):  # pylint: disable=arguments-differ
         context['project'] = self.project
         context['active_builds'] = active_builds
         context['versions'] = self._get_versions(self.project)
-        context['build_qs'] = self.get_queryset()
+
+        builds = self.get_queryset()
+        if settings.RTD_EXT_THEME_ENABLED:
```

Comment:
I think we can just always run it through the filter, no need for the setting. It just won't get filtered, if the user doesn't have the GET args.

Reply:
That was what I originally had here, but updated on feedback already. I think the conditional is fine, we'll test this in the beta on production data.